build.gradle

```diff
@@ -29,9 +29,9 @@ sourceSets {
     java {
       compileClasspath += main.output
       runtimeClasspath += main.output
-      srcDir file('src/functionalTest/java')
+      srcDir file('src/integrationTest/java')
     }
-    resources.srcDir file('src/functionalTest/resources')
+    resources.srcDir file('src/integrationTest/resources')
   }
 
   integrationTest {
```

Comment:
we shoudl remove integration task if we're renaming it to functional, it has no purpose otherwise